packages/host/app/components/ai-assistant/new-session-button.gts

```diff
@@ -40,41 +35,76 @@ export default class NewSessionButton extends Component<Signature> {
     this.selectedOptions = new Set(this.selectedOptions);
   }
 
+  @action
+  handleCreateNewSession(event: MouseEvent) {
+    // Check if Shift key is pressed
+    if (event.shiftKey) {
+      event.preventDefault();
+      this.showMenu = !this.showMenu;
+      return;
+    }
+
+    // Normal click behavior
+    const addSameSkills = this.selectedOptions.has('Add Same Skills');
+    this.args.onCreateNewSession(addSameSkills);
```

Comment:
Normal click should always create the new session as if no special options have been selected. The path with the options should only happen if you shift-click and then click the Create button in the modal.

Reply:
Updated 

https://github.com/user-attachments/assets/ffd83abe-8e92-441f-80b8-4946b0fe5d7c